lib/pengine/rules.c

```diff
@@ -684,8 +694,18 @@ pe_eval_expr(xmlNode *rule, pe_rule_eval_data_t *rule_data, crm_time_t *next_cha
     return passed;
 }
 
+/*!
+ * \brief Evaluate rules
```

Comment:
It doesn't really matter (clear from name) but maybe "Evaluate a rule's subexpression" or something like that. We use "Evaluate rules" in the next commit for `pe_eval_rules()`.

Documentation is tricky due to the number of layers -- e.g., `pe_evaluate_rules()`, `pe_eval_rules()`, `pe_eval_expr()`, `pe_eval_subexpr()`, `pe_eval_<expr_type>()`...

Reply:
Oh yeah, I meant to come back to that. Added commit on top.